Readme.md

```diff
@@ -117,6 +117,10 @@ log.info('this will be written to /tmp/debug.stream.out and /tmp/info.stream.out
 log.fatal('this will be written to /tmp/debug.stream.out, /tmp/info.stream.out and /tmp/fatal.stream.out')
 ```
 
+`opts` multistream options object. Available options are:
+
++ `dedupe`: Set this to `true` to send logs only to the stream with the higher level. Default: `false`
```

Comment:
A short example here would help. Easiest to understand is probably the error logs case that prompted this PR.

Reply:
@jsumners done